src/Components/Components/ref/Microsoft.AspNetCore.Components.netstandard2.0.cs

```diff
@@ -3,6 +3,33 @@
 
 namespace Microsoft.AspNetCore.Components
 {
+    public delegate void AuthenticationStateChangedHandler(System.Threading.Tasks.Task<Microsoft.AspNetCore.Components.IAuthenticationState> newAuthenticationStateTask);
+    public partial class AuthenticationStateProvider : Microsoft.AspNetCore.Components.ComponentBase, System.IDisposable
```

Comment:
Should this be sealed? Is there anything reasonable to do with it by subclassing?

Reply:
I didn't think we had a way to mark `.razor`-generated classes as `sealed`. Please correct me if I'm wrong!

Reply:
Yeah, I think you're right 😢 we don't have that.